File: Assets/scripts/exit.js
#pragma strict
var mySkin:GUISkin;
function Start () {

}

function Update () {

}

function OnGUI(){
	GUI.skin=mySkin;
	if(GUI.Button(new Rect(190,280,100,40),"",GUI.skin.GetStyle("exitButton"))){
		Application.Quit();
	}

}

File: Assets/scripts/UIScript/C/Exit.cs
using UnityEngine;
using System.Collections;

public class Exit : MonoBehaviour {
    public GUISkin mySkin;
	// Use this for initialization
	void Start () {
       
	}
	
	// Update is called once per frame
	void Update () {
      
	}

    void OnGUI() {
        GUI.skin = mySkin;
        if (GUI.Button(new Rect(190f, 280f, 100f, 40f), "", GUI.skin.GetStyle("exitButton")))
        {
            Application.Quit();
        }
    }
}
